docs/02-app/02-api-reference/05-next-config-js/images.mdx

```diff
@@ -225,3 +226,16 @@ export default function supabaseLoader({ src, width, quality }) {
   return url.href
 }
 ```
+
+### PixelBin
+
+```js
+// Doc (Resize): https://www.pixelbin.io/docs/transformations/basic/resize/#width-w
+// Doc (Optimise): https://www.pixelbin.io/docs/optimizations/quality/#image-quality-when-delivering
+export default function pixelBinLoader({ src, width, quality }) {
+  return src.replace(
+    `/original/`,
+    `/t.resize(w:${width})~t.compress(q:${quality || 75})/`
```

Comment:
This should also have `f_auto`

https://www.pixelbin.io/docs/optimizations/format/

Reply:
This can be configured from PixelBin console panel if user want it to be best format to deliver.

Scenario 1: BEST FORMAT ENABLE and FORCE BEST FORMAT DISABLE - here user have choice to handle best format with query param `f_auto` and user can choose to not use best format wherever they don't want

Scenario 1: BEST FORMAT ENABLE and FORCE BEST FORMAT ENABLE - Always best format will serve by [PixelBin](https://www.pixelbin.io/docs/optimizations/format/)

So to respect the users choice, I am not suggesting to add it in example PixelBin loader.

Reply:
What scenario would you not want auto format?

This is the default behavior for the built-in Image Optimization API so I would expect the same when recommending a 3rd party loader.

Reply:
In specific to ML transformation like erasing background, upscaling media, there will be quality loss. But considering general use case of most of users to serve just images. Adding `f_auto` query param and updating PR.

Reply:
Updated example code of PixelBin image loader